.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.1.0
+    rev: v4.3.0
```

Comment:
other things that can be included here:

      - id: check-case-conflict
      - id: check-toml
      - id: detect-private-key

Reply:
Added all of them. Good suggestions!